### ISSUE TITLE
Datasets for retraining

### ISSUE THREAD

**Issue by FJGEODEV** [LTTM/LSR#2]
Hi, thank you for your great work! I have some questions about it.

For source pretraining, GTA5 with resnet101, as you shown as an example, which cityscapes datasets are required? I downloaded gtFine_trainvaltest.zip, but looks like it requires "leftImg8bit_trainvaltest.zip" and "leftImg8bit_trainextra.zip", do I miss any datasets?

In the command, `-data_root_path (path to dataset) --list_path (path to split lists)` should be pointed to GTA5 data and list, `--target_root_path (path to dataset) --target_list_path (path to split lists) ` should point to cityscapes data and list. Just want to confirm. 

Look forward to your response. Thanks.

**Comment by barbafrank** [comment]
Hi, sorry for the delay in the reply.
For the standard training (i.e. using the cityscapes 'train' split, not 'train_plus' or 'train_extra') you only need the first zip (leftImg8bit_trainvaltest.zip).
Your suppositions are correct, the data_root and list_path refer to GTA, the ones with prefix target refer to cityscapes.
In case you missed them, you can find the splits used here: https://github.com/LTTM/LSR/tree/main/datasets/splits

**Comment by cht619** [comment]
NTHU dataset train dataset no label?

**Comment by barbafrank** [comment]
Hello.
I am not sure why you are asking about the dataset in this repo and not directly to its authors, but yes - NTHU does not have a labeled training set.
This code is for an Unsupervised Domain Adaptation strategy, meaning we don't use/need labeled target samples during the adaptation process. The NTHU dataset was designed for the UDA task and only provides a few labeled validation samples to compute the final accuracy metrics.
